src/common/components/Embeds/index.tsx

```diff
@@ -19,6 +20,8 @@ export const renderEmbedForUrl = ({ url }: { url: string }) => {
     return <NounsBuildEmbed url={url} />
   } else if (url.includes('paragraph.xyz') || url.includes('pgrph.xyz')) {
     return <ParagraphXyzEmbed url={url} />
+  } else if (url.startsWith('https://modprotocol.org')) {
```

Comment:
use OpenGraphImage for all remaining urls and not just for urls that start with https://modprotocol.org